tests/__init__.py

```diff
@@ -359,3 +361,64 @@ def assert_url_equal(url1, url2):
     assert_equal(parts1.hostname, parts2.hostname)
     assert_equal(parts1.port, parts2.port)
     assert_equal(parse_qs(parts1.query), parse_qs(parts2.query))
+
+
+class HTTPStubberException(Exception):
+    pass
+
+
+class RawResponse(BytesIO):
+    def stream(self, **kwargs):
+        contents = self.read()
+        while contents:
+            yield contents
+            contents = self.read()
+
+
+class ClientHTTPStubber(object):
+    def __init__(self, client):
+        self.reset()
+        self._client = client
+
+    def reset(self):
+        self.requests = []
+        self.responses = []
+
+    def create_response(self, url=None, status=200, headers=None, body=None):
```

Comment:
Why not use the verbs from the original stubber (`add_response()`)?  It seems confusing to have something called `create_response` without actually returning anything.

Reply:
I'm fine with this, I originally had an `add_response` and `create_response`.